2016-06-03.md

```diff
@@ -0,0 +1,34 @@
+# Improving Rust, Letting Go of Python, Open Finances, and More
+
+## Multiple language support
+
+### Ramping up Rust
+We've long been intrigued by the Rust Language, and have lately been pushing forward for a more native Rust support experience, including easy deploy from the `t2` command line tool.
+
+You can see the state of the project at the [tessel-rust](https://github.com/tessel/tessel-rust) repo. If you're testing it out, please let us know what you think! Issues, PRs, and conversation in the #rust-lang channel of the [Tessel Slack](//tessel-slack.herokuapp.com) are all welcome.
```

Comment:
I don't know enough about @johnnyman727 's work (particularly the cross-compliation Docker server) so I can't comment here.